volumes/add.go

```diff
@@ -0,0 +1,42 @@
+package volumes
+
+import (
+	"context"
+	"os"
+
+	"github.com/unweave/cli/config"
+	"github.com/unweave/cli/ui"
+	"github.com/unweave/unweave/api/types"
+)
+
+var rest = config.InitUnweaveClient()
```

Comment:
This won't work. This get's called before the config is initialized. Which means the token won't be set and you'll get an unauthenticated error.